b2sdk/b2http.py

```diff
@@ -124,9 +124,12 @@ def _translate_and_retry(fcn, try_count, post_params=None):
         except B2Error as e:
             if not e.should_retry_http():
                 raise
-            if hasattr(e, 'retry_after_seconds') and e.retry_after_seconds is not None:
+            exc_message_format = 'Retrying again after :%s seconds'
```

Comment:
```python
            if e.retry_after_seconds is not None:
                sleep_duration = e.retry_after_seconds
                sleep_reason = 'server asked us to'
            else:
                sleep_duration = wait_time
                sleep_reason = 'that is what the default exponential backoff is'
            logger.info('Pausing thread for %i seconds because %s', sleep_duration, sleep_reason)
            time.sleep(sleep_duration)
            wait_time *= 1.5
```